Documentation/boostrap-new-os.md

```diff
@@ -0,0 +1,140 @@
+After the build succeeds, list the contents of the `bin/Product/Linux.x64.Release\.nuget\pkg` directory and note the version number e.g. in the `Microsoft.NETCore.Runtime.CoreCLR` package filename. So for example, the filename can be `Microsoft.NETCore.Runtime.CoreCLR.2.0.2-servicing-25708-0.nupkg`. The version is `2.0.2-servicing-25708-0`. It will be needed in the next step.
+### Building CoreFX
+Now that we have built the CoreCLR, we can build the CoreFX. Since CoreFX depends on CoreCLR packages, we need to pass the location of the packages built in CoreCLR as an additional nuget source using the `/p:OverridePackageSource=xxxx` option. Also, the corefx build needs to know the version of the coreclr packages so that it can get proper nuget packages. The previous paragraph on CoreCLR build describes how to get the version. 
+Change the current directory to the corefx directory, edit the `dependencies.props` file and replace the version in the `<CoreClrPackageVersion>` node by the version of your coreclr build.
+Then run the following command.
+```bash
+DOTNET_TOOL_DIR=/your/path/to/bootstrap/dotnetcli ./build.sh -release -runtimeos=mynewrid -stripSymbols -SkipTests=true -- /p:OverridePackageSource=../coreclr/bin/Product/Linux.x64.Release/.nuget/pkg/ /p:PortableBuild=false
+```
+If the build completes ok, it would generate nuget `.nupkg` package files into `bin/packages/Release`. If it fails, investigate and fix the problems and run the build command again.
+Now you need to find the version of the nuget packages that were just built. List files in the `bin/packages/Release` folder and note the version number in the `Microsoft.Private.CoreFx.NETCoreApp` package file. So for example, if the filename is `Microsoft.Private.CoreFx.NETCoreApp.4.4.0-preview1-25304-0.nupkg`, the version is `4.4.0-preview1-25304-0`. You will need it in the next step.
+### Building Core-Setup
+The core-setup build depends on the nuget packages that were just built in coreclr and corefx. Since the overriding of package source can only be a single directory, we first create a folder where we copy the nuget packages from both coreclr and corefx. 
+Change the current directory to the parent directory of coreclr, corefx and core-setup
+```bash
+mkdir combinednugets
+cp coreclr/bin/Product/Linux.x64.Release\.nuget\pkg\*.nupkg combinednugets
+cp corefx/bin/packages/Release/*.nupkg combinednugets
+```
+Now change the current directory to the core-setup directory, edit the `dependencies.props` file and replace the version in the `<MicrosoftPrivateCoreFxNETCoreAppPackageVersion>` node by the version of your corefx build and the version in the `<MicrosoftNETCoreRuntimeCoreCLRPackageVersion>` node by the version of your coreclr build.
+Then build core-setup using the following command:
+```bash
+DOTNET_TOOL_DIR=/your/path/to/bootstrap/dotnetcli ./build.sh -portable=false -release -stripSymbols -DistroRid=mynewrid-x64 -os=Linux -- /p:OverridePackageSource=../combinednugets/
```

Comment:
The `-DistroRid` option was removed and replaced with `/p:OutputRid=mynewrid-x64`

Reply:
This change hasn't made its way to master yet.  But it is in PR - https://github.com/dotnet/core-setup/pull/3291